.gitattributes

```diff
@@ -1,3 +1,239 @@
+### Header #########################################################################################
+
+# Author: Florian Bernd
```

Comment:
Is this needed?

Reply:
Ah yes, I copied it from one of my personal templates. We can of course remove the header 😋